platform/src/constants.rs

```diff
@@ -13,3 +13,8 @@ pub mod yield_id {
     pub const NO_WAIT: u32 = 0;
     pub const WAIT: u32 = 1;
 }
+
+pub mod exit_id {
+    pub const TERMINATE: u32 = 0;
+    pub const RESTART: u32 = 0;
+}
```

Comment:
nitpick: We should probably order `exit_id` before `syscall_class`, as these are in alphabetical order.